src/components/gameover/GameOver.tsx

```diff
@@ -44,17 +44,16 @@ const GameOver = (props: GameOverProps) => {
     }
   }, [gameState.storableScore]);
 
-  const participate = (name: string, email: string, subscribe: boolean) => {
-    if (name.trim() === '') {
-      console.log('Name must be filled in');
-      return;
-    }
-
-    if (email.trim() === '') {
-      console.log('E-mail must be filled in');
-      return;
+  const getEmail = (name: string) => {
+    for (const entry of gameState.scoreList) {
+      if (entry.name === name) {
+        return entry.email;
+      }
     }
+    return '';
+  };
```

Comment:
This could be moved into ScorePage where it is actually used, since the gameState is a globally available state using the context GameContext (see how it is set up in this component).
We can do this later